### ISSUE TITLE
Collision shape impossible to move when keyframed in one animation

### ISSUE THREAD

**Issue by JeanMakesGames** [godotengine/godot#75179]
### Godot version

Godot 4.0, Godot 4.0.1

### System information

Windows 11, i5, surface pro 8, 8go RAM, Iris X

### Issue description

When you keyframe the position of a collision shape 2D, or the one of its parent (here an Area2D), you can't change anymore the position of that collision shape or Area2D in an other animation, here's a video with commentary to show what my problem is:

https://user-images.githubusercontent.com/88930512/226573333-941d7bed-eb81-4d27-9764-23155f9a65f7.mp4

### Steps to reproduce

try to key a collision in one animation, then key it on an other one

### Minimal reproduction project

N/A

**Comment by TokageItLab** [comment]
Minimal reproduction project is needed. And isn't AnimationTree enabled? While AnimationTree is enabled, AnimationTree updates take precedence.

**Comment by JeanMakesGames** [comment]
Yes animation tree is enabled :) I was using the same technique before in godot 3.5 and that was not giving any issue with the animation tree, has it changed with godot 4?

For the minimal reproduction, how should I do? Upload my project so far?

**Comment by TokageItLab** [comment]
In Godot 4, properties that may be blended for blending consistency are always blended as initial values (or Reset animation values) even if the blendamount for that animation is 0.

This is a specification, not a bug. Just could add this to the documentation.

> For the minimal reproduction, how should I do? Upload my project so far?

Perhaps now that we know the cause and it is no longer needed.

**Comment by JeanMakesGames** [comment]
So yes just to confirm, when the animation tree is not active, then I can move the collision shape and key it in the animation player. So that's not a bug, it's just user error here as i was suspecting myself, but like i wasn't finding any reference to that i didn't know, sorry for that. but thanks for your answer, it definitely helps solved that issue :)
